### DISCUSSION TITLE
Warnings, Errors log, diagnostic messages... and how to fix it...

### DISCUSSION THREAD

**Discussion by vuongquoc** [LycheeOrg/Lychee#2267] [Q&A]
Hi,
It's me again... 🍡 

I'm checking Logs and have these warnings, errors log... 
So do i need to fix all this? and how can i?

```
Warning: App\Metadata\Versions\GitHubVersion::isGit:127 Could not read /...hidden.../.git/HEAD
Error: App\Actions\HoneyPot\BasePipe::throwNotFound:40 The route fonts/roboto-v29-400.woff2 could not be found.
```

And i have these notice in my Diagnostic
```
    Info: Found 3 small2x that could be generated.
         You can use `php artisan lychee:generate_thumbs small2x 3` to generate them.
    Info: Found 3 medium that could be generated.
         You can use `php artisan lychee:generate_thumbs medium 3` to generate them.
    Info: Found 4 medium2x that could be generated.
         You can use `php artisan lychee:generate_thumbs medium2x 4` to generate them.
```
but i have no idea how to fix it... please help me clean up these notice...

Thanks.

**Comment by ildyria** [comment]
> Error: App\Actions\HoneyPot\BasePipe::throwNotFound:40 The route fonts/roboto-v29-400.woff2 could not be found.

Seems like for some reason the font roboto-v29-400 is not found in your installation (maybe we forgot to provide it? :( ).

> Warning: App\Metadata\Versions\GitHubVersion::isGit:127 Could not read /...hidden.../.git/HEAD

Nothing to really worry about, just make sure that the `.git` folder is readable (if it exists) by your web user.

For the small2x etc... You can go into `/path/to/lychee` and run the mentioned command. It should generate the missing files.

**Reply by vuongquoc** [reply]
Hi @ildyria 

- For the fonts. can i have it, please?
- I don't see the .git folder in my hosting.

and for this
---
For the small2x etc... You can go into /path/to/lychee and run the mentioned command. It should generate the missing files. <<<
---
what app should i use to run that command? my godaddy don't have docker (assume that it's a needed app)

Thanks.

**Reply by ildyria** [reply]
https://github.com/LycheeOrg/Lychee/blob/master/resources/fonts/roboto-v29-400.woff2

you need to access via ssh to be able to execute that command (or wait for us to implement a button for it).

**Comment by vuongquoc** [comment]
> https://github.com/LycheeOrg/Lychee/blob/master/resources/fonts/roboto-v29-400.woff2
this font already exist in the folder /resources/fonts/ but still got the error log... @_@

**Comment by ildyria** [comment]
That link was for you to be able to download the file, not where it should be placed.
It seems that the generation of the front is having a tiny miss on that.
Will need to investigate further. That being said, I wouldn't worry about it.

**Reply by vuongquoc** [reply]
Thanks Sir.

I'm just a noob so i'll wait.

Best regards.,

Quoc

**Comment by ildyria** [comment]
I think this has been resolved. :)
